deap/tools/support.py

```diff
@@ -523,7 +523,16 @@ def update(self, population):
             self.insert(population[0])
 
         for ind in population:
-            if ind.fitness > self[-1].fitness or len(self) < self.maxsize:
+            if isinstance(ind.fitness, (tuple, list)):
```

Comment:
Are you sure about this? Doesn't lexicase fitnesses  hinherit from base.Fitness? I'm sure that a base.Fitness is an object, not a list.

Otherwise, see line 529 comment.